src/Language/Haskell/TH/Lens.hs

```diff
@@ -1738,52 +1668,29 @@ _Interruptible
 -- |
 -- @
 -- _InlineP :: 'Prism'' 'Pragma' ('Name', 'Inline', 'RuleMatch', 'Phases') -- template-haskell-2.8+
--- _InlineP :: 'Prism'' 'Pragma' ('Name', 'Inline')                    -- Earlier versions
 -- @
-#if MIN_VERSION_template_haskell(2,8,0)
 _InlineP :: Prism' Pragma (Name, Inline, RuleMatch, Phases)
 _InlineP
   = prism' reviewer remitter
   where
       reviewer (x, y, z, w) = InlineP x y z w
       remitter (InlineP x y z w) = Just (x, y, z, w)
       remitter _ = Nothing
-#else
-_InlineP :: Prism' Pragma (Name, InlineSpec)
-_InlineP
-  = prism' reviewer remitter
-  where
-      reviewer (x, y) = InlineP x y
-      remitter (InlineP x y) = Just (x, y)
-      remitter _ = Nothing
-#endif
 
 -- |
 -- @
 -- _SpecialiseP :: 'Prism'' 'Pragma' ('Name', 'Type', 'Maybe' 'Inline', 'Phases') -- template-haskell-2.8+
```

Comment:
Now that the API for `_SpecialiseP` is uniform across all supported versions of `template-haskell`, this Haddock comment is completely redundant. I'd favor removing it entirely.